src/eins-hwinfo.c

```diff
@@ -103,6 +101,59 @@
 #define COMPUTER_HWINFO_TYPE_STRING "(" RAMINFO_TYPE_STRING \
   ROOTFS_SPACE_TYPE_STRING "@" CPUINFO_ARRAY_TYPE_STRING ")"
 
+/* 24 hours */
+#define RECORD_COMPUTER_HWINFO_INTERVAL_SECONDS (60u * 60u * 24u)
+
+/* The path of a file to hold next record time. */
+#define RECORD_TIME_FILE_PATH INSTRUMENTATION_CACHE_DIR "record_time"
+
+static gboolean
+get_next_record_time (guint *record, GError **error)
+{
+  gsize len = sizeof (guint);
+
+  return g_file_get_contents (RECORD_TIME_FILE_PATH,
+		              (gchar **)&record,
+			      &len,
+			      error);
+}
```

Comment:
I would prefer for this file to be a `.ini`-style GKeyFile, like the existing `/var/cache/eos-metrics-instrumentation/persistent-tallies`. It's easier to inspect by hand and involves less scary casting of pointers to read.

Reply:
Something like:

```
g_autoptr(GKeyFile) kf = g_key_file_new ();

g_key_file_load_from_file (kf, RECORD_TIME_FILE_PATH, G_KEY_FILE_NONE, NULL);
return g_key_file_get_int64 (kf, "hwinfo", "next-record-time", NULL);
```

Reply:
ok